beginner_source/blitz/tensor_tutorial.py

```diff
@@ -94,6 +105,30 @@
 # Tensor Operations
 # ~~~~~~~~~~~~~~~~~
 #
+# **Check for a tensor**
+# 
+# We can check whether an object in python is a tensor or not. Typically, is_tensor function checks 
+# and is_storage function checkes whether an object is stored as tensor object.
```

Comment:
> is_storage function checkes whether an object is stored as tensor object

This doesn't sound correct to me.

Reply:
I have changed it, referring to the doc in 
https://pytorch.org/docs/stable/generated/torch.is_storage.html